app/controllers/api/revoke_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Api
+  class RevokeController < BaseController
+    def validate_method_action(_method_name, action_name)
+      super(:put, action_name)
+    end
+
+    def target_user
+      userid = params[:userid]
+      if userid
+        user_object = User.find_by(:userid => userid)
+        raise BadRequestError, "Unable to find user #{userid}" unless user_object
+
+        filtered_user = Rbac::Filterer.filtered_object(user_object)
+        raise ForbiddenError, "The user is not authorized for this task or item." unless filtered_user
```

Comment:
we might also need to check if target user is not the current user, then current_user.super_admin_user? should be true unless we introduce a new entitlement for this, in which case something liek current_user.role_allows?(:identifier => 'revoke_user_sessions') could be introduced.

Reply:
@abellotti I updated PR as you suggested:
- I added check if target user is not the current user 
- I added feature `revoke_user_sessions` in core for this operation
- I refactored it a bit.

can you take a look please ? thanks